src/models/converters/tests/get_address_ex_from_any_source.rs

```diff
@@ -0,0 +1,60 @@
+use crate::models::converters::get_address_ex_from_any_source;
+use crate::models::service::addresses::AddressEx;
+use crate::providers::info::*;
+
+#[rocket::async_test]
+async fn get_address_info_address_diff_than_safe() {
+    let address = "0x1234";
+    let safe = "0x4321";
+
+    let mut mock_info_provider = MockInfoProvider::new();
+    mock_info_provider
+        .expect_address_ex_from_any_source()
+        .times(1)
+        .return_once(move |_| {
+            Ok(AddressEx {
+                value: address.to_string(),
+                name: Some("".to_string()),
+                logo_url: None,
+            })
+        });
+
+    let expected = AddressEx {
+        value: address.to_string(),
+        name: Some("".to_string()),
+        logo_url: None,
+    };
+
+    let actual = get_address_ex_from_any_source(safe, address, &mut mock_info_provider).await;
+
+    assert_eq!(expected, actual);
+}
+
+#[rocket::async_test]
+async fn get_address_info_address_diff_than_safe_error() {
+    let address = "0x1234";
+    let safe = "0x4321";
+
+    let mut mock_info_provider = MockInfoProvider::new();
+    mock_info_provider
+        .expect_address_ex_from_any_source()
+        .times(1)
+        .return_once(move |_| bail!("No address info"));
+
+    let actual = get_address_ex_from_any_source(safe, address, &mut mock_info_provider).await;
+    assert_eq!(AddressEx::address_only(address), actual);
+}
+
+#[rocket::async_test]
+async fn get_address_info_address_equal_to_safe() {
+    let address = "0x1234";
+    let safe = "0x1234";
+
+    let mut mock_info_provider = MockInfoProvider::new();
+    mock_info_provider
+        .expect_address_ex_from_contracts()
+        .times(0);
+
+    let actual = get_address_ex_from_any_source(safe, address, &mut mock_info_provider).await;
+    assert_eq!(AddressEx::address_only(address), actual);
+}
```

Comment:
ah ok this is just a rename from `src/models/converters/tests/get_address_info.rs`. Not terribly important but I find this https://git-scm.com/docs/git-mv preferable, if commit history matters more (in our case we squash anyhow)

Reply:
If it would be a pure move this would automatically be detected from github. As the content of the file also changed (all method got renamed) it was not a simple move